src/es/mangacrab/src/eu/kanade/tachiyomi/extension/es/mangacrab/MangaCrab.kt

```diff
@@ -62,4 +62,8 @@ class MangaCrab :
             else -> element.attr("abs:src")
         }
     }
+
+    companion object {
+        val URL_REGEX = """^https?://(?:[A-Za-z0-9-]+\.)+[A-Za-z]{2,}(?::\d+)?(?:/[^\s?#]*)?(?:\?[^\s#]*)?(?:#\S*)?$""".toRegex()
```

Comment:
Seems complex and hides what you're actually looking for.

Instead, consider:
```css
img:not([src])
```

And just getting the first value of any attribute that starts with `http`.

... or is valid according to `HttpUrl`:
```kt
val imageAbsUrl = element.attributes().firstOrNull { it.toHttpUrlOrNull() != null }?.value
```